src/mod.ts

```diff
@@ -22,12 +23,86 @@ export const run = async function (functionDir: string) {
     return potentialFunctionFiles;
   });
 
-  // The CLI expects a JSON payload to be output to stdout
-  // This is formalized in the `run` hook of the CLI/SDK Tech Spec:
-  // https://corp.quip.com/0gDvAsqoaaYE/Proposal-CLI-SDK-Interface#temp:C:fOC1991c5aec8994d0db01d26260
-  console.log(JSON.stringify(resp || {}));
+  return JSON.stringify(resp || {});
```

Comment:
Maybe we could move the `JSON.stringify` wrapper up into the http server layer, leaving this to be agnostic to how it's serialized for transport?

Reply:
oh yeah I like that good idea will do!